ui/src/helpers/extract-initials.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+export function extractInitials(fullName: string | undefined): string {
+  // Remove non-alphanumeric characters.
+  const strippedName = (fullName ?? '').replace(/[^a-zA-Z0-9 ]/g, '');
+
+  // Split words, omitting empty strings.
+  const words = strippedName.split(' ').filter(Boolean);
+  if (words.length == 0) return '??';
```

Comment:
I wonder if we should rather return undefined here and handle what's displayed for them where this is used? I'd say we never want to show `??` as the avatar.

Reply:
> I wonder if we should rather return undefined here and handle what's displayed for them where this is used?

Yes.

> I'd say we never want to show `??` as the avatar.

Here I disagree. I believe having that circle with `??` is better than an empty circle or no circle if initials could not be determined.